frontend/src/routes/(authenticated)/admin/+page.svelte

```diff
@@ -143,14 +147,11 @@
         </div>
       </AdminTabs>
       <div class="mt-4">
-        <FilterBar
-          debounce
-          loading={$loadingUsers}
-          searchKey="userSearch"
-          filterKeys={userFilterKeys}
```

Comment:
I believe the UserFilter class needs to be passed the `filterKeys` attribute just like FilterBar did, and also that you need to update the `userFilterKeys` value on line 49 to include 'usersICreated'. The reason why this needs to go on line 49 of the admin dashboard, rather than in the UserFilter module, is because `userFilterKeys` is used in the `loadingUsers` logic on line 53, which allows the admin dashboard to display a loading indicator while the GraphQL query is carried out.